test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -612,6 +615,18 @@ public ExecutorPlus executorFor(int verbId)
         return Verb.fromId(verbId).stage.executor();
     }
 
+    @Nullable
+    private DurationSpec startupTimeout()
```

Comment:
> 9) JVM Dtest now supports startup timeouts, this is to avoid issues where startup will take infinity (due to bugs) causing CI to throw away the logs.

when accord hits a bug bootstrap is blocked forever, which then means we get 0 logs in CI as we just get "timeout"... we *need* to bound our startup logic in order to get CI to tell us we are broken and why